app/views/pages/index.html.erb

```diff
@@ -29,20 +29,34 @@
 
 <% explanation_petitions do |actioned| %>
   <section aria-labelledby="response-threshold-heading" id="petitions-with-response">
-    <div class="threshold-panel response-threshold">
-      <h2 id="response-threshold-heading"><%= Site.formatted_threshold_for_response %></h2>
-      <p>If a petition gets <%= Site.formatted_threshold_for_response %> signatures, the government will&nbsp;respond</p>
-    </div>
+    <% if actioned[:with_response][:count].zero? %>
+      <div class="threshold-panel response-threshold">
+        <h2 id="response-threshold-heading"><%= Site.formatted_threshold_for_response %></h2>
```

Comment:
I know :(
I wanted to use `link_to_unless` but it doesn't accept the url and then the `a` tag contents in a `do end` block